add_connector.md

```diff
@@ -281,6 +394,38 @@ Don’t forget to add logs lines in appropriate places.
 Refer to other connector code for trait implementations. Mostly the rust compiler will guide you to do it easily.
 Feel free to connect with us in case of any queries and if you want to confirm the status mapping.
 
+### **Set the currency Unit**
+The `get_currency_unit` function, part of the ConnectorCommon trait, enables connectors to specify their accepted currency unit as either `Base` or `Minor`. For instance, Paypal designates its currency in the Base unit, whereas Hyperswitch processes amounts in the minor unit. If a connector accepts amounts in the base unit, conversion is required, as illustrated.
```

Comment:
This can be improved by citing examples of what Base and Minor units are. Something like:

```text
[...] Paypal designates its currency in the base unit (for example, USD), whereas Hyperswitch processes amounts in the minor unit (for example, cents).
```